docs/source/guides.md

```diff
@@ -0,0 +1,42 @@
+# How-to Guides
+
+This section contains guides of how to perform common functions with the api
+
+## Discover devices
+
+```{eval-rst}
+.. automodule:: kasa.discover
+```
+
+## Connect without discovery
+
+```{eval-rst}
+.. automodule:: kasa.deviceconfig
+```
+
+## Get Energy Consumption and Usage Statistics
+
+```{note}
+    In order to use the helper methods to calculate the statistics correctly, your devices need to have correct time set.
+    The devices use NTP and public servers from `NTP Pool Project <https://www.ntppool.org/>`_ to synchronize their time.
+```
+
+### Energy Consumption
+
+The availability of energy consumption sensors depend on the device.
+While most of the bulbs support it, only specific switches (e.g., HS110) or strips (e.g., HS300) support it.
+You can use {attr}`~SmartDevice.has_emeter` to check for the availability.
```

Comment:
Hmm, this should probably be `Device.has_emeter` or show an example how to use the modules interface? Although we do not currently have a common API for emeter...

Reply:
Done